<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Preetijaiswal-IT" failures="3" tests="3" name="SeleniumTutorialPractice.dataProvideTest" time="0.000" errors="0" timestamp="8 Feb 2022 16:48:11 GMT">
  <testcase classname="SeleniumTutorialPractice.dataProvideTest" name="test2" time="0.000">
    <failure message="
DataProvider should be static: public java.lang.Object[][] SeleniumTutorialPractice.DataProvideDemo.dataset1(java.lang.reflect.Method)" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
DataProvider should be static: public java.lang.Object[][] SeleniumTutorialPractice.DataProvideDemo.dataset1(java.lang.reflect.Method)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:328)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:262)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:414)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1389)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1081)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1186)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="SeleniumTutorialPractice.dataProvideTest" name="test" time="0.000">
    <failure message="
DataProvider should be static: public java.lang.Object[][] SeleniumTutorialPractice.DataProvideDemo.dataset1(java.lang.reflect.Method)" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
DataProvider should be static: public java.lang.Object[][] SeleniumTutorialPractice.DataProvideDemo.dataset1(java.lang.reflect.Method)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:328)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:262)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:414)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1389)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1081)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1186)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase classname="SeleniumTutorialPractice.dataProvideTest" name="test1" time="0.000">
    <failure message="
DataProvider should be static: public java.lang.Object[][] SeleniumTutorialPractice.DataProvideDemo.dataset1(java.lang.reflect.Method)" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
DataProvider should be static: public java.lang.Object[][] SeleniumTutorialPractice.DataProvideDemo.dataset1(java.lang.reflect.Method)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:328)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:262)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:414)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1389)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1081)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1186)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- SeleniumTutorialPractice.dataProvideTest -->
